templates/layout-with-agenda-submenu.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+{% endblock %}
```

Comment:
Do we need this block if we don't override anything?

Reply:
Removed redundant `parent()` call.